package.json

```diff
@@ -1,29 +1,31 @@
 {
   "name": "yamlify-object",
-  "version": "0.6.0",
+  "version": "1.0.0",
   "description": "Stringify object/array with yaml syntax",
   "main": "./dist/index.js",
```

Comment:
I think here you should add "types": "dist.index.d.ts" as noted here https://www.tsmean.com/articles/how-to-write-a-typescript-library/